sdk/core/azure-core/src/main/java/com/azure/core/spatial/GeometryPosition.java

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.spatial;
+
+/**
+ * Represents a geometric position.
+ */
+public final class GeometryPosition {
+    private final double longitude;
+    private final double latitude;
+
+    private final Double altitude;
+
+    /**
+     * Constructs a geometric position.
+     *
+     * @param longitude Longitudinal position.
+     * @param latitude Latitudinal position.
+     */
+    public GeometryPosition(double longitude, double latitude) {
+        this(longitude, latitude, null);
+    }
+
+    /**
+     * Constructs a geometric position.
+     *
+     * @param longitude Longitudinal position.
+     * @param latitude Latitudinal position.
+     * @param altitude Altitude position.
+     */
+    public GeometryPosition(double longitude, double latitude, Double altitude) {
+        this.longitude = longitude;
+        this.latitude = latitude;
+        this.altitude = altitude;
+    }
+
+    /**
+     * The longitudinal position of the geometric position.
+     *
+     * @return The longitudinal position.
+     */
+    public double getLongitude() {
+        return longitude;
+    }
+
+    /**
+     * The latitudinal position of the geometric position.
+     *
+     * @return The latitudinal position.
+     */
+    public double getLatitude() {
+        return latitude;
+    }
+
+    /**
+     * The altitude of the geometric position.
+     *
+     * @return The altitude.
+     */
+    public Double getAltitude() {
```

Comment:
I understand why you have Double type in these places, but I'm not super fond of it. I would wonder if we should have a boolean check that clarifies if it has been set or not.

Reply:
I can add APIs such `hasAltitude`, `hasMinAltitude`, `hasMaxAltitude` for the scenarios where we have null-able geoposition data. Do you feel that these add enough benefit for having additional APIs?